third_party/proto/tendermint/libs/kv/types.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 package tendermint.libs.kv;
-option  go_package = "github.com/tendermint/tendermint/libs/kv";
```

Comment:
why do we need this file?

Reply:
I think it should just be deleted?

Reply:
ah this got overwritten somehow. this should stay where it is because its still used in the abci.